packages/logging/logging.go

```diff
@@ -5,13 +5,15 @@ import (
 	"os"
 	"time"
 
+	"github.com/AplaProject/go-apla/packages/conf"
+
 	"github.com/AplaProject/go-apla/packages/model"
 	"github.com/AplaProject/go-apla/packages/utils"
 )
 
 // WriteSelectiveLog is writing selective log
 func WriteSelectiveLog(text interface{}) error {
-	if *utils.LogLevel == "DEBUG" {
+	if conf.Config.LogLevel == "DEBUG" {
```

Comment:
i thought that this function was removed. Anyway, WriteSelectiveLog is no longer in use

Reply:
removed,